plugins/modules/proxmox_kvm.py

```diff
@@ -1284,7 +1288,7 @@ def main():
             else:
                 module.exit_json(changed=False, vmid=vmid, msg="VM {0} is already on {1}".format(vmid, node))
         except Exception as e:
-            module.fail_json(vmid=vmid, msg='Unable to migrate VM {0} from {1} to {2}: {3}'.format(vmid, vm_node, node, e))
+            module.fail_json(vmid=vmid, msg='Unable to migrate VM {0}: {3}'.format(vmid, e))
```

Comment:
`vm_node` and `node` could be None when an exception is thrown, so I deleted them.

Reply:
In case they are sometimes not `None`, they should be kept, otherwise that information isn't shown anymore (which might make debugging harder).